### ISSUE TITLE
Pronunciation audio button doesnt work

### ISSUE THREAD

**Issue by teapotonmars** [TriangleLeft/Flashcards#14]
When clicking on a work in the main word list, the button for the pronunciation isn't working

**Comment by teapotonmars** [comment]
(At least, for Hungarian)

**Comment by alekseikurnosenko** [comment]
@emmaliddell Yep, doesn't work for me either. The link to retrieve audio file returns AccessDenied. Probably due to the fact that language is still in beta.
Is it working for you if you try it from "Words" section on website?

**Comment by ShapeShifter499** [comment]
I'm having this issue with Duolingo Japanese and your app

**Comment by ShapeShifter499** [comment]
Did some searching online, seems like the words section was removed. No wonder why I couldn't find it on the desktop site.

**Comment by licaon-kter** [comment]
#23 ?

**Comment by ShapeShifter499** [comment]
@licaon-kter sorry I missed that issue ticket. I might try the official Duolingo tinycards app but I do try to use open source where possible.
